### PR TITLE
Add grid alternatives related to DKLAT, DKMSL and GLLMSL

### PR DESCRIPTION
**Pull request by kbevers**
Goes along with https://github.com/OSGeo/PROJ-data/pull/121

### CODE REVIEW

File: data/sql/grid_alternatives.sql [hunk 1]
```diff
@@ -144,6 +144,11 @@ VALUES
 ('dvr90_2002.tif','dk_sdfi_dvr90_2002.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dvr90_2002.tif',1,1,NULL),
 ('dvr90_2013.tif','dk_sdfi_dvr90_2013.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dvr90_2013.tif',1,1,NULL),
 ('dvr90_2023.tif','dk_sdfi_dvr90_2023.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dvr90_2023.tif',1,1,NULL),
+('dklat_2022.tif','dk_sdfi_dklat_2022.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dklat_2022.tif',1,1,NULL),
+('dklat_2023.tif','dk_sdfi_dklat_2023.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dklat_2023.tif',1,1,NULL),
+('dkmsl_2022.tif','dk_sdfi_dkmsl_2022.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dkmsl_2022.tif',1,1,NULL),
+('dkmsl_2023.tif','dk_sdfi_dkmsl_2023.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dkmsl_2023.tif',1,1,NULL),
```
Comment (rouault) [inline data/sql/grid_alternatives.sql]: AFAICS, those 4 should be geoid_like

File: data/sql/grid_alternatives.sql [hunk 2]
```diff
@@ -144,6 +144,11 @@ VALUES
...
+('gllmsl_2022.tif','dk_sdfi_gllmsl_2022.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_gllmsl_2022.tif',1,1,NULL),
```
Comment (rouault) [inline data/sql/grid_alternatives.sql]: and that one:
```suggestion
('gllmsl_2022.tif','dk_sdfi_gllmsl_2022.tif',NULL,'GTiff','vgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_gllmsl_2022.tif',1,1,NULL),
```

File: data/sql/grid_alternatives.sql (hunk 1 above)
Comment (rouault) [inline data/sql/grid_alternatives.sql]:
```suggestion
('dklat_2022.tif','dk_sdfi_dklat_2022.tif',NULL,'GTiff','geoid_like',0,NULL,'https://cdn.proj.org/dk_sdfi_dklat_2022.tif',1,1,NULL),
('dklat_2023.tif','dk_sdfi_dklat_2023.tif',NULL,'GTiff','geoid_like',0,NULL,'https://cdn.proj.org/dk_sdfi_dklat_2023.tif',1,1,NULL),
('dkmsl_2022.tif','dk_sdfi_dkmsl_2022.tif',NULL,'GTiff','geoid_like',0,NULL,'https://cdn.proj.org/dk_sdfi_dkmsl_2022.tif',1,1,NULL),
('dkmsl_2023.tif','dk_sdfi_dkmsl_2023.tif',NULL,'GTiff','geoid_like',0,NULL,'https://cdn.proj.org/dk_sdfi_dkmsl_2023.tif',1,1,NULL),
```

File: data/sql/grid_alternatives.sql (hunk 2 above)
Comment (jjimenezshaw) [inline data/sql/grid_alternatives.sql]: I am confused. Are they `hgridshift`? some seem to be geoid models, and the last one a vertica shift, right?

Comment (kbevers) [reply data/sql/grid_alternatives.sql]: Your confusion is correct. They are all `vgridshift`'s. I'll fix it right away!

File: data/sql/grid_alternatives.sql (hunk 1 above)
Comment (kbevers) [reply data/sql/grid_alternatives.sql]: Yeah. Actually I just copied and modified the above DVR90-lines, so those need adjusting as well

**Comment by kbevers** [comment]
Thanks for you input @jjimenezshaw and @rouault - you have good eyes!

**Comment by kbevers** [comment]
Not sure if the fuzzing error is related to my changes or not. Guessing it is not

**Comment by rouault** [comment]
> Guessing it is not

was a temporary network issue with a script fetched by libtiff. Seems to be fixed  now

**Pull request merged**